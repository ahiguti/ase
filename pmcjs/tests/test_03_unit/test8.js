
Perl.Use("NewTest");
var nt = Perl.Functions["NewTest"];
nt.newtest();

function Foobar(x) {
  this.propx = "this-is-propx";
  this.methody = function(a, b) {
    return a * b + x;
  }
}
nt.newtest_foobar();

var Foo = {
  Bar : function(x) {
    this.propy = "this-is-propy";
    this.methodz = function(a, b) {
      return a * b * x;
    }
  }
};
nt.newtest_fb();

