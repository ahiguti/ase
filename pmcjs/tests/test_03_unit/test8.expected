len=5
arr 10
arr 20
arr 30
arr 40
arr 50
x=this-is-propx y=20003
z=12500
